### ISSUE TITLE
Impossible to login in externals.io

### ISSUE THREAD

**Issue by jmquarck** [mnapoli/externals#121]
When I tried to login in externals.io with my github account, top left link, I was redirected to the the typical Oauth authorization panel. But the authorize button is disabled, so this panel is a dead end.

Is this intended, as a consequence of the fact that my github username is not either a member of the externals, not of the internals members list? Or is it a bug (as I think)?

**Comment by mnapoli** [comment]
Thank you for reporting this, this is definitely a bug! Externals is open to everyone.

Would you be able to post a screenshot here of the GitHub page?

**Comment by jmquarck** [comment]
I repeated the operation of clicking on the button in Chrome in my mobile phone, and it worked. I mean, I was redirected to the common Oauth2 panel to authorize a 3-party application using my github username, and then the green button to grant this permission was clickable, so I clicked on and it worked.

Therefore, I am not able to say whether the trouble with my first attemp  in my Ubuntu 18 + Chrome browser was just temporary, or not.

I guess you might close this issue unless additional reports happen.
